style.css

```diff
@@ -124,35 +176,403 @@ body {
 
 .social-section {
   margin-right: 80px;
+  display: grid;
 }
 
 /* social media icons----------------------------------------------------------------------- */
+.Workssection1 {
+  border-radius: 8px;
+}
+
+/* Card Works */
+.cardBack {
+  margin-top: 114px;
+  margin-bottom: 104px;
+  margin-left: 24px;
+  margin-right: 24px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  height: 596px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
 
 .cardBack01 {
+  background-image: url('./image/Mobile-Main_3.png');
   background-repeat: no-repeat;
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 16px;
+  width: 375px;
   height: 568px;
   border: 0 solid #dfe1e6;
   border-bottom-left-radius: 50px;
 }
 
+.cardBack_02 {
+  margin-top: 114px;
+  margin-bottom: 104px;
+  margin-left: 24px;
+  margin-right: 24px;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  align-items: center;
+  padding: 16px;
+  height: 596px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [ ] **As suggested by the previous reviewer too**, I would also recommend to use **CSS Flexbox** to lay out an individual card to meet the project requirements,:100: You may kindly replace ```display: grid;``` with ```display:flex``` and add a ```flex-direction: column;``` to order items form top to bottom :+1: :+1: 

Kindly check [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for more